frontend/src/views/NewChannelView.vue

```diff
@@ -1,7 +1,128 @@
 <template>
 	<div class="page_box_wrapper">
 		<div class="page_box">
-			<h1>This is a new channel page</h1>
+			<h1>NEW CHANNEL</h1>
+			<div class="c_block c_form_group">
+				<form
+					method="Post"
+					action=""
+					class="c_block c_form_group"
+					@submit.prevent="createNewChannel(createNewChannelForm)"
```

Comment:
2 nested elements with the same classes, won't do much in this specific case
you can remove the div surrounding the form